c2corg_api/views/user_mailinglists.py

```diff
@@ -0,0 +1,82 @@
+from c2corg_api import DBSession
+from c2corg_api.models.user import User
+from c2corg_api.models.mailinglist import Mailinglist
+from c2corg_api.views import cors_policy, restricted_json_view, \
+    restricted_view
+from c2corg_common.attributes import mailinglists
+from cornice.resource import resource
+
+
+def validate_mailinglist_statuses(request, **kwargs):
+    tested = []
+    for ml in request.validated:
```

Comment:
Is the data in `validated`? If you are not using a schema or another validator that has written data to `validated`, you must get your data from `request.GET` or `request.POST`.